### DISCUSSION TITLE
[feature request] Disallow "+1" comments (or at least suppress notifications)

### DISCUSSION THREAD

**Discussion by lobsterkatie** [community/community#53075] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

It would be great to be able to prevent people from adding "+1" comments, or at least allow subscribers to not be notified of them. 

I'm subscribed to various issues and discussions about feature requests which interest me, but my notifications inbox is constantly getting spammed with updates where it turns out the only new thing is yet one more person having said "+1". I'd love to find a way to make it stop, without having to unsubscribe from meaningful updates.

UPDATE: Turns out this is even in the [GH code of conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct#best-practices-for-maintaining-a-strong-community) (scroll down to the very bottom of the section):

> Rather than posting “+1” or “Agreed”, use the upvote button.

(Thanks to @vintagesucks for pointing this out)

P.S. Forgive me if this is a duplicate. Exactly because of all of those spammy comments, it's really hard to search for "+1".

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by jordanbtucker** [comment]
+1

**Reply by lobsterkatie** [reply]
🤦🏻‍♀️
